pkg/compare/compare.go

```diff
@@ -433,27 +434,28 @@ func (o *Options) Run() error {
 		return fmt.Errorf("failed to collect resources: %w", err)
 	}
 	r.IgnoreErrors(func(err error) bool {
-		return containOnly(err, []error{MultipleMatches{}, UnknownMatch{}})
+		return containOnly(err, []error{MultipleMatches{}, UnknownMatch{}, MergeError{}})
```

Comment:
you don't have to put  the mergeerror inside the containOny because the error isn't joined, only the corealtor returns a  joined errors that need this behavior, but i guess it should work like this also .
so up to you, but just for notice

Reply:
It has a clause for non-joined errors, I thought it would be good re-use that as to not make more branches and vars than needed.